lumicks/pylake/kymotracker/kymoline.py

```diff
@@ -1,6 +1,67 @@
 import numpy as np
 
 
+def export_kymolines_to_csv(filename, kymo_lines, dt, dx, delimiter, sampling_width):
+    """Export kymolines to a csv file.
+
+    Parameters
+    ----------
+    filename : str
+        Filename to output kymolines to.
+    kymo_lines : KymoLines[pylake.KymoLine]
+        List of kymograph traces to export.
+    dt : float
+        Calibration for the time axis.
+    dx : float
+        Calibration for the coordinate axis.
+    delimiter : str
+        Which delimiter to use in the csv file.
+    sampling_width : int or None
+        When supplied, this will sample the source image around the kymograph line and export the summed intensity with
+        the image. The value indicates the number of pixels in either direction to sum over.
+    """
+    if not kymo_lines:
+        raise RuntimeError("No kymolines to export")
+
+    idx = np.hstack([idx * np.ones(len(line.time_idx)) for idx, line in enumerate(kymo_lines)])
```

Comment:
Couple minor suggestions:
- I think `len(line)` is clearer than `len(line.time_idx)`
- I always forget about this function, but using `np.full()` might be a bit more clear
```suggestion

    idx = np.hstack([np.full(len(line), idx) for idx, line in enumerate(kymo_lines)])
```

Reply:
Thanks. That one wasn't on my radar, but it is indeed nicer.